apps/labs/posts/doctesting-for-pydata-libraries.md

```diff
@@ -0,0 +1,90 @@
+---
+title: 'Doctesting for PyData Libraries'
+author: [Sheila-nk]
+published: September 30, 2023
+description: 'The journey of a PyData Newbie'
+category: [Doctests]
```

Comment:
```suggestion
category: [Developer workflows]
```

We haven't defined a `Doctests` category for the blog, so suggesting `Developer workflows` - which I think is a good fit here.